src/app_service/service/keycardV2/service.nim

```diff
@@ -1,67 +1,242 @@
-import NimQml, json, chronicles, strutils
-# import keycard_go
+import NimQml, json, chronicles, strutils, random, json_serialization
+import keycard_go
 import app/global/global_singleton
 import app/core/eventemitter
 import app/core/tasks/[qt, threadpool]
+import app_service/service/keycard/service as old_keycard_service
+import ../../../backend/response_type
+import ../../../constants as status_const
+import ./dto
+
+proc callRPC*(rpcCounter: int, methodName: string, params: JsonNode = %*{}): string  =
+    var request = %*{
+      "id": rpcCounter,
+      "method": "keycard." & methodName,
+      "params": %*[ params ],
+    }
+    var response = keycard_go.keycardCallRPC($request)
+    debug "callRPC", request, response
```

Comment:
Time to drop this I think. It will log all mnemonics, PINs and other wonderful information